Assets/AoTTG 2.unity

```diff
@@ -115,7 +115,7 @@ Transform:
   m_LocalScale: {x: 0.0027777802, y: 0.002777778, z: 0.00277778}
   m_Children: []
   m_Father: {fileID: 0}
-  m_RootOrder: 3
```

Comment:
Did you change anything to the main scene? If so, rather undo all changes here as it seems like Unity is doing something weird here:
![image](https://user-images.githubusercontent.com/22615133/155394501-0bf1b7d1-9b86-4ef6-b7c0-2282e7a14775.png)

Reply:
re import all assets, it's not the brach, since I re imported all the assests before pushing

Reply:
maybe not, i'm getting it as well

Reply:
I believe I fixed it